File: spring-boot-rest/src/main/scala/com/stulsoft/poc/rest/SomeService.scala
/*
 * Copyright (c) 2018. Yuriy Stul
 */

package com.stulsoft.poc.rest

import javax.inject.Inject
import org.slf4j.LoggerFactory
import org.springframework.core.env.{ConfigurableEnvironment, PropertySource}
import org.springframework.stereotype.Service

/**
  * @author Yuriy Stul
  */
@Service
class SomeService @Inject()(env: ConfigurableEnvironment) {
  private val logger = LoggerFactory.getLogger(classOf[SomeService])

  def foo(): Unit = {
    val propertySources = env.getPropertySources.iterator()
    while (propertySources.hasNext) {
      logger.info(s"Property source: ${propertySources.next().getName}")
    }
  }
}

File: pjunit5/spring-boot-junit5/src/main/java/com/stulsoft/poc/pjunit5/spring/boot/SomeService.java
/*
 * Copyright (c) 2018. Yuriy Stul
 */

package com.stulsoft.poc.pjunit5.spring.boot;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

/**
 * @author Yuriy Stul
 */
@Component
public class SomeService {
    private static final Logger logger = LoggerFactory.getLogger(SomeService.class);

    public void doServiceWork(){
        logger.info("==>doServiceWork");
        logger.info("<==doServiceWork");
    }
}